eslint-config-5minds/package.json

```diff
@@ -3,7 +3,7 @@
   "publishConfig": {
     "registry": "https://registry.npmjs.org/"
   },
-  "version": "0.2.0",
+  "version": "0.3.0",
```

Comment:
Why did you increment the MINOR version while raising the PATCH version at `tslint-eslint-config-5minds`? Can you explain your intention?